cogstat/cogstat.py

```diff
@@ -909,6 +909,151 @@ def explore_variable(self, var_name, frequencies=True, central_value=0.0):
         result_list.append(text_result)
         return cs_util.convert_output(result_list)
 
+
+    def reliability_internal(self, var_names, reverse_items=None):
+        """
+        Calculate internal consistency reliability using Cronbach's alpha and it's confidence interval,
+        as well as item-rest correlations and their confidence intervals.
+
+        Parameters
+        ----------
+        var_names : list of str
+            Names of the variables or items.
+        reverse_items : list of str
+            Names of reverse coded variables or items.
+
+        Returns
+        -------
+        list of str and image
+            Analysis results in HTML format
+        """
+
+        meas_levels = [self.data_measlevs[var_name] for var_name in var_names]
+
+        title = '<cs_h1>' + _('Internal consistency reliability') + '</cs_h1>'
+        title += '\n' + _('Reliability of item(s): ') + ', '.join('%s (%s)' % (var, meas)
```

Comment:
```suggestion
        title += '\n' + _('Reliability of items: ') + ', '.join('%s (%s)' % (var, meas)
```
I assume that a single variable cannot be used for this analysis.